spec/models/work_version_spec.rb

```diff
@@ -59,6 +59,14 @@
 
       it_behaves_like 'a resource with orderable creators'
     end
+
+    describe '#file_version_memberships' do
+      let(:resource) { create(:work_version, :with_files, file_count: 2) }
+
+      it 'orders the files by title in ascending order' do
+        expect(resource.file_version_memberships.map(&:title)).to eq(['image-1.png', 'image-2.png'])
```

Comment:
You need to rename the files and verify that they get sorted correctly. Something like:
``` ruby
resource.file_version_memberships[0].update(title: 'image-4.png')
resource.file_version_memberships[1].update(title: 'image-3.png')
```
ought to do the trick.